c/termination-crafted/2Nested_true-termination_true-valid-memsafety.c

```diff
@@ -18,10 +18,10 @@ int main()
 	y = __VERIFIER_nondet_int();
 
   // prevent overflow: x + y*(y+1)/2 <= 2^31-1
-  __VERIFIER_assume(x<=65534);
-  __VERIFIER_assume(y<=65534);
+  if(!(x<=65534)) return 0;
```

Comment:
This change is ok, but I do not understand why it is needed.
Maybe this is better for tools that do not support `_VERIFIER_assume`.